ios/engine/KMEI/KeymanEngine/Classes/Manager.swift

```diff
@@ -1274,7 +1266,7 @@ public class Manager: NSObject, HTTPDownloadDelegate, UIGestureRecognizerDelegat
   }
   
   /// - Returns: The current state for a lexical model
-  public func stateForLexicalModel(withID lexicalModelID: String) -> LexicalModelState {
+  public func stateForLexicalModel(withID lexicalModelID: String) -> KeyboardState {
```

Comment:
I assume the possible state variants are all identical?

In the future, we may want to rename this something like `ResourceState`, as it's a little more general for referring to our keyboard + language modeling resources.  (I started using that term in some of the Android strings and new types.)

Reply:
Yes they are identical.
ResourceState is a good idea.
I noticed the lack of circle-i icon, as well, but only for the last selected model. If you select MTNT, it can't be Info'ed, but Sencoten can be. If Sencoten is selected, MTNT can be but Sencoten cannot be.
I think I know what's going on and will fix it now.

Reply:
The info view is always accessible, now, no matter whether it is the current lexical model or not.